save-cloud-common/build.gradle.kts

```diff
@@ -36,7 +36,10 @@ kotlin {
 
     sourceSets {
         sourceSets.all {
-            languageSettings.optIn("kotlin.RequiresOptIn")
+            languageSettings.apply {
+                optIn("kotlin.RequiresOptIn")
+                optIn("kotlin.js.ExperimentalJsExport")
```

Comment:
Guess it should be in `jsMain` only

Reply:
actually not, `JsExport` is applicable for common and shared code. Like `Dto` classes for examle